From a854dc7108cdb70b5f58a1232868c74ad3620c99 Mon Sep 17 00:00:00 2001
From: Vinson Lee <vlee@freedesktop.org>
Date: Sat, 20 Aug 2016 06:02:27 +0000
Subject: [PATCH] video_splliter: Add XCB flags for panoramix.

This patch fixes this build error.

video_splitter/.libs/libpanoramix_plugin_la-panoramix.o: In function `CountMonitors':
modules/video_splitter/panoramix.c:352: undefined reference to `xcb_connect'
modules/video_splitter/panoramix.c:354: undefined reference to `xcb_connection_has_error'
modules/video_splitter/panoramix.c:357: undefined reference to `xcb_get_setup'
modules/video_splitter/panoramix.c:359: undefined reference to `xcb_setup_roots_iterator'
modules/video_splitter/panoramix.c:360: undefined reference to `xcb_screen_next'
modules/video_splitter/panoramix.c:407: undefined reference to `xcb_disconnect'
modules/video_splitter/panoramix.c:407: undefined reference to `xcb_disconnect'
modules/video_splitter/panoramix.c:407: undefined reference to `xcb_disconnect'

Signed-off-by: Vinson Lee <vlee@freedesktop.org>
Signed-off-by: Jean-Baptiste Kempf <jb@videolan.org>
---
 modules/video_splitter/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/video_splitter/Makefile.am b/modules/video_splitter/Makefile.am
index da32c52..ffeb94e 100644
--- a/modules/video_splitter/Makefile.am
+++ b/modules/video_splitter/Makefile.am
@@ -15,8 +15,8 @@ if HAVE_WIN32_DESKTOP
 splitter_LTLIBRARIES += libpanoramix_plugin.la
 else
 if HAVE_XCB_RANDR
-libpanoramix_plugin_la_CFLAGS += $(XCB_RANDR_CFLAGS)
-libpanoramix_plugin_la_LIBADD += $(XCB_RANDR_LIBS)
+libpanoramix_plugin_la_CFLAGS += $(XCB_RANDR_CFLAGS) $(XCB_CFLAGS)
+libpanoramix_plugin_la_LIBADD += $(XCB_RANDR_LIBS) $(XCB_LIBS)
 splitter_LTLIBRARIES += libpanoramix_plugin.la
 endif
 endif
-- 
2.9.3

